packages/client/tests/functional/raw-queries/typed-results/prisma/_schema.ts

```diff
@@ -12,7 +13,7 @@ export default testMatrix.setupSchema(({ provider }) => {
     }
     
     model TestModel {
-      id     Int       @id
+      id    ${idForProvider(provider)}
```

Comment:
Since I added tests and I didn't want to make the tests rely on the id I tweaked this, it does not change the test, not in what's tested.